<ui version="4.0" >
 <class>NotificationWidget</class>
 <widget class="QWidget" name="NotificationWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="balloonGroupBox" >
     <property name="title" >
      <string>バルーン通知</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="notifyFavoriteCheckBox" >
          <property name="text" >
           <string>お気に入り</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_2" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>新着</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_4" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>詳細変更あり</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>通知するお気に入りチャンネルの最低スコア</string>
          </property>
          <property name="indent" >
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="minimumScoreSpinBox" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="minimum" >
           <number>0</number>
          </property>
          <property name="maximum" >
           <number>999</number>
          </property>
          <property name="value" >
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="soundGroupBox" >
     <property name="title" >
      <string>サウンド通知</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="playSoundCheckBox" >
          <property name="text" >
           <string>ファイルを再生</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="soundFileEdit" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectFileButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>選択...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playSoundButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>♪</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>再生可能ファイルは WAV と OGG です。(Windows は WAV のみ)</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="indent" >
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>118</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>playSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>81</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel" >
     <x>91</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectFileButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>59</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel" >
     <x>445</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>soundFileEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>96</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel" >
     <x>212</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>playSoundButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>51</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel" >
     <x>531</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>soundGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectFileButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>142</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel" >
     <x>445</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifyFavoriteCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>128</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel" >
     <x>129</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifyFavoriteCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minimumScoreSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>179</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel" >
     <x>368</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
